CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Added
 
+- AWS SDK add `rpc.system` attribute. (#3617)
```

Comment:
Please use the full package import path instead of just saying "AWS SDK". That way users can search the changelog to find changes related to the package they are using.